.github/workflows/build-img-ghrunner-test.yaml

```diff
@@ -0,0 +1,50 @@
+name: build-img-ghrunner-test
+
+on:
+  push:
+    branches: [ main ]
```

Comment:
I think we can remove push, as I set up rule for require rebase on PRs...meaning just before merge we already tested with latest changes in code.  WDYT?

Reply:
yeah we can remove it in that case